cumulusci/robotframework/SalesforcePlaywright.py

```diff
@@ -0,0 +1,84 @@
+from Browser import SupportedBrowsers
+
+from cumulusci.robotframework.BaseLibrary import BaseLibrary
+
+
+class SalesforcePlaywright(BaseLibrary):
+    ROBOT_LIBRARY_SCOPE = "Suite"
+
+    def __init__(self):
+        super().__init__()
+        self._browser = None
+
+    @property
+    def browser(self):
+        if self._browser is None:
+            self._browser = self.builtin.get_library_instance("Browser")
+        return self._browser
+
+    def delete_records_and_close_browser(self):
+        """This will close all open browser windows and then delete
+        all records that were created with the Salesforce API during
+        this testing session.
+        """
+        self.browser.close_browser("ALL")
+        self.salesforce_api.delete_session_records()
+
+    def open_test_browser(self, size=None, useralias=None, recordVideo=None):
```

Comment:
```suggestion
    def open_test_browser(self, size=None, useralias=None, record_video=None):
```

Any reason we're not using snake case?